test/Bugs/randombug.baseline

```diff
@@ -63,10 +63,10 @@ Test case 30
 Passed
 Test case 31
 Error
-	at func2 (randombug.js:287:9)

```

Comment:
Just to chime in about `TrimStackTracePath`: a few days ago I saw in Full that when unit tests run with `-BVT` (by default), we strip away paths from stack traces inside runtime. Maybe we can consolidate these in future. There are probably more flags for related/similar purposes: `-BaselineMode` (and `ExtendedErrorStackForTestHost`? not sure).